docs/src/main/sphinx/installation/cli.rst

```diff
@@ -46,36 +193,58 @@ The recommended TLS implementation is to use a globally trusted certificate. In
 this case, no other options are necessary, since the JVM running the CLI
 recognizes these certificates.
 
-Use ``--help`` to see information about specifying the keystore, truststore, and
-other authentication details as required.
+Use the options from the following table to further configure TLS and
+certificate usage:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--insecure``
+    - Skip certificate validation when connecting with TLS/HTTPS (should only be
+      used for debugging).
+  * - ``--keystore-path``
+    - The location of the Java Keystore file that contains the certificate of
+      the server to connect with TLS.
+  * - ``--keystore-password``
+    - The password for the keystore. This must match the password you specified
+      when creating the keystore.
+  * - ``--keystore-type``
+    - Keystore type.
```

Comment:
Maybe link to https://docs.oracle.com/en/java/javase/11/docs/specs/security/standard-names.html#keystore-types for SOME of the allowed values. Note that the values may differ by whatever JDK (and security providers) user has installed.

Reply:
Same for `truststore-type`.

Reply:
@mosabua please suggest wording to indicate that the values will vary by whatever JDK people use, so consult the similar docs for their JDK.

Reply:
I would prefer to delay this whole addition since this is quite a bit more complicated and should maybe be handled in more detail in our JKS and PEM file page. I also dont know what is truly supported (e.g. do pem files work like on the server).  We could just link to our TLS page and the certificate section but it is not helpful enough at the moment either. Can we handle this all in a follow up ticket?

Reply:
Sure, please create one to track.

Reply:
bump @mosabua, please track this via an issue so that we don't forget to follow-up.